cumulus/parachains/runtimes/assets/asset-hub-kusama/src/lib.rs

```diff
@@ -0,0 +1,1480 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
```

Comment:
@muharem shouldn't this file be removed from here? Kusama-related?

Reply:
already gone from master